src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -312,35 +312,25 @@ void medToolBox::handleDisplayError(int error)
 {
     switch (error)
     {
-    case medAbstractProcess::PIXEL_TYPE:
-        displayPixelTypeError();
+    case medAbstractProcess::PIXEL_TYPE:   //! Handle volume errors: pixel type
+        displayMessageError("Pixel type not yet implemented");
         break;
-    case medAbstractProcess::DIMENSION:
-        displayDataDimensionError();
+    case medAbstractProcess::DIMENSION_3D: //! Handle volume errors: dimension
+        displayMessageError("This toolbox is designed to be used with 3D volumes");
         break;
-    case medAbstractProcess::MESH_TYPE:
-        displayMeshTypeError();
+    case medAbstractProcess::DIMENSION_4D: //! Handle volume errors: dimension
+        displayMessageError("This toolbox is designed to be used with 4D volumes");
+        break;
+    case medAbstractProcess::MESH_TYPE:    //! Handle mesh errors: data type
+        displayMessageError("This toolbox is designed to be used with meshes");
+        break;
+    default:
```

Comment:
If the code calls ```handleDisplayError``` with an error other than the ones above, we would still want some message to be displayed. Something like  "This action failed (undefined error)".